parm/soca/soca_ecen_stage.yaml.j2

```diff
@@ -0,0 +1,19 @@
+mkdir:
+- '{{ DATA }}/ens'
+link:
+{% for imem in range(1,NMEM_ENS+1) %}
+    {% for domain in ['ocean', 'ice'] %}
+- [ "{{ DATA }}/../ensdata/ens/{{ domain }}.{{ imem | string }}.nc", '{{ DATA }}/ens/' ]
+    {% endfor %}
+{% endfor %}
+copy:
+{% for imem in range(1,NMEM_ENS+1) %}
+    {% set memchar = 'mem%03d' | format(imem) %}
+    {% set tmpl_dict = ({ '${ROTDIR}': ROTDIR,
+                          '${RUN}': RUN,
+                          '${YMD}': previous_cycle | to_YMD,
+                          '${HH}': previous_cycle | strftime('%H'),
+                          '${MEMDIR}': memchar }) %}
+- ['{{ COM_ICE_RESTART_TMPL | replace_tmpl(tmpl_dict) }}/{{ cice_rst_date }}.cice_model.res.nc', '{{ DATA }}/ens/cice_model.res.{{ imem | string }}.nc']
+- ['{{ COM_ICE_RESTART_TMPL | replace_tmpl(tmpl_dict) }}/{{ cice_rst_date }}.cice_model.res.nc', '{{ DATA }}/ens/cice_model.res.output.{{ imem | string }}.nc']
```

Comment:
Not necessary to address here, more of a comment. @guillaumevernieres I think we can cut down on time a little by removing this copy and changing cice restart that was copied above in place. (I added an extra copy for debugging at some point, and forgot to remove it).